script.js

```diff
@@ -0,0 +1,11 @@
+const menuToggle = document.querySelector('.menu-toggle input');
+const menulink = document.querySelectorAll('nav ul li a');
+const nav = document.querySelector('nav ul');
+menuToggle.addEventListener('click', () => {
+  nav.classList.toggle('slide');
+  console.log('hello');
```

Comment:
- [ ] Kindly follow the [JavaScript best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) as mentioned in the project requirements. Do not commit `console.log` to your repo. It’s ugly, it kills performance and it can make confidential data be visible to anyone using the browser tools to look at your website.